index.js

```diff
@@ -124,13 +124,19 @@ function sortPackageJson(packageJson) {
 }
 module.exports = sortPackageJson;
 module.exports.sortPackageJson = sortPackageJson;
-if (require.main === module) {
-  var fs = require('fs');
-  var packageJsonPath = process.cwd() + '/package.json';
-  var packageJson = fs.readFileSync(packageJsonPath, 'utf8');
+
+function processFile(filePath) {
+  var packageJson = fs.readFileSync(filePath, 'utf8');
   var sorted = sortPackageJson(packageJson);
   if (sorted !== packageJson) {
-    fs.writeFileSync(packageJsonPath, sorted, 'utf8');
-    console.log('Ok, your package.json is sorted');
+      fs.writeFileSync(filePath, sorted, 'utf8');
+      console.log(filePath + ' is sorted!');
   }
 }
+
+if (require.main === module) {
+  var fs = require('fs');
```

Comment:
I don't like that we require fs here, and up above on line L132 use fs. This is relying on lexical scope hoisting which - although it works - I dislike. Perhaps just make `processFile` an anonymous function to `forEach`?

Otherwise this code looks 👍.

Reply:
Totally agree - this was a result of refactoring without looking at the whole picture.